### PR TITLE
DSN-2907 grid container padding changed to 1.5rem at med size

### PR DESCRIPTION
**Pull request by kelseypaone**

### CODE REVIEW

File: src/_variables.scss [hunk 1]
```diff
@@ -23,7 +23,7 @@ $gutter-width: 1rem !default;
 
 /// Default grid outer margin
 /// @type Length
-$outer-margin: 2rem !default;
+$outer-margin: 1.5rem !default;
```
Comment (jordangehman) [inline src/_variables.scss]: If this is updated, are columns and row offset affected? @davegomez

**Review by jordangehman — COMMENTED** [review]
FTR I personally think `1.5rem` still seems large for mobile screens, but 0-768px is a large span. BTW 48em is the small breakpoint, not medium as indicated in the commit message.

File: src/_variables.scss (hunk 1 above)
Comment (kelseypaone) [reply src/_variables.scss]: no, this variable is only used on container styling

**Comment by kelseypaone** [comment]
**New standards set for container and container-fluid**
base padding size: 1rem
at sm (min-width: 48rem): 1.5rem
at md (min-width: 64rem): 2rem

**Review by jordangehman — APPROVED** [review]

**Comment by davegomez** [comment]
I approve the changes but the PR must be resend because of the following issues:

* The aw-grid titles doesn't need the ticket names (`DSN-2907`) because this repository is public.
* Use the [Git Flow](http://nvie.com/posts/a-successful-git-branching-model/) approach to name and manage the branches.
* The source branch must be `develop` instead of `master` and the target branch for the PR must be `develop` as well.
* All the changes must be done in the **SCSS** files, the distribution files (**CSS**) are generated by the build commands and can not be manually modified.
* After apply all your changes in the **SCSS** code build the distribution files and check the result is what you expect in the [`aw-grid.css`](https://github.com/aweber/aw-grid/blob/master/dist/aw-grid.css) file before publishing the PR.

Read the [project documentation](https://github.com/aweber/aw-grid/blob/develop/README.md) to understand how to work with the project's commands and how the project is structured.

PD: close this PR before opening the new one.

**Pull request closed without merging**